File: src/main/java/Main/ItemClasses/Consumable.java
package Main.ItemClasses;

public class Consumable extends Item {
    public Consumable(String name, String description, int weight, int valueInGold, int amount) {
        super(name, description, weight, valueInGold, amount);
        this.iconName = "Loaf of Bread.png";
    }

    public Consumable() {
        super();
        this.iconName = "Loaf of Bread.png";
    }
}

File: src/main/kotlin/main/itemClasses/Consumable.kt
package main.itemClasses

import kotlinx.serialization.Serializable

@Serializable
open class Consumable() : Item() {
    init {
        standardIconName = "Loaf of Bread.png"
    }

    constructor(name: String, description: String, weight: Int, valueInGold: Int, amount: Int) : this() {
        this.name = name
        this.description = description
        this.weight = weight
        this.valueInGold = valueInGold
        this.amount = amount
    }
}
